docs/content/docs/start/tutorial.md

```diff
@@ -96,78 +106,90 @@ async function start() {
     location.hash = id;
   }
 }
-start().catch((error) => console.error(error));
 
+start().catch((error) => console.error(error));
 ```
 
 ## Write the dice view
 
-The Fluid Framework is view framework agnostic and works well with React, Vue, Angular and web components. This example uses standard HTML/DOM methods to render a view. You can see examples of the previously mentioned frameworks in the [FluidExamples repo](https://github.com/microsoft/FluidExamples/tree/main/multi-framework-diceroller).
-
-### Start with a static view
+The Fluid Framework is agnostic about view frameworks and it works well with React, Vue, Angular and web components. This example uses standard HTML/DOM methods to render a view. You can see examples of the previously mentioned frameworks in the [FluidExamples repo](https://github.com/microsoft/FluidExamples/tree/main/multi-framework-diceroller).
 
-It is simplest to create the view using local data without Fluid, then add Fluid by changing some key pieces of the app. This tutorial uses this approach.
+The `renderDiceRoller` function runs only when the container is created or loaded. It appends the `diceTemplate` to the passed in HTML element, and creates a working dice roller with a random dice value each time the "Roll" button is clicked on a client. 
 
-The `renderDiceRoller` function appends the `diceTemplate` to the passed in HTML element, and creates a working dice roller with a random dice value each time the "Roll" button is clicked. The `diceMap` will be used in the next few steps.
 
 ```js
 const diceTemplate = document.createElement("template");
 
 diceTemplate.innerHTML = `
+  <style>
+    .wrapper { text-align: center }
+    .dice { font-size: 200px }
+    .roll { font-size: 50px;}
+  </style>
   <div class="wrapper">
     <div class="dice"></div>
     <button class="roll"> Roll </button>
   </div>
 `
-function renderDiceRoller(diceMap, elem) {
-    elem.appendChild(diceTemplate.content.cloneNode(true));
+const renderDiceRoller = (diceMap, elem) => {
+    elem.appendChild(template.content.cloneNode(true));
+
     const rollButton = elem.querySelector(".roll");
     const dice = elem.querySelector(".dice");
 
-    rollButton.onclick = () => updateDice(Math.floor(Math.random() * 6)+1);
-
-    const updateDice = (value) => {
-        // Unicode 0x2680-0x2685 are the sides of a die (⚀⚁⚂⚃⚄⚅).
-        dice.textContent = String.fromCodePoint(0x267f + value);
-    };
-    updateDice(1);
+    /* REMAINDER OF THE FUNCTION IS DESCRIBED BELOW */
```

Comment:
It seems little bit odd to describe the remaining code outside of the context of `renderDiceRoller` without finally landing to how it all stiches together.

Reply:
> It seems little bit odd to describe the remaining code outside of the context of `renderDiceRoller` without finally landing to how it all stiches together.

To me that's just one of the oddities of a tutorial that doesn't walk readers through creating the app. Instead, it gives readers a link in the first sentence to a sample and then **talks** readers through code. We should consider rewriting this someday as a true tutorial.